tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Azure/Functions/AzureFunctionsCommon.cs

```diff
@@ -443,17 +440,18 @@ private static bool TryParseJson<T>(object? jsonObj, [NotNullWhen(true)] out T?
             }
         }
 
-        private static bool AreAllContextsIdentical(List<SpanContext> contexts)
+        private static bool AreAllContextsIdentical(List<PropagationContext> contexts)
```

Comment:
We should probably rename this method now because the name and signature imply it's comparing all the `PropagationContext` items, but it's only comparing the `SpanContext` property in each and ignoring baggage.

Maybe `AreAllSpanContextsIdentical()`? And add a comment to explain that it ignore baggage?